extensions/Cake.ExtractLicenses.yml

```diff
@@ -0,0 +1,24 @@
+Type: Addin
+Name: Cake.ExtractLicenses
+NuGet: Cake.ExtractLicenses
+Assemblies:
+- /**/Cake.ExtractLicenses.dll
+Repository: https://github.com/wgnf/liz.git
+ProjectUrl: https://github.com/wgnf/liz/
+Author: wgnf
+Description: >-
+  The Cake-Addin of the Liz-Tool - which helps to extract license-information from your solution/project
+              aimed on a fast and correct process.
+Categories:
+- license
+- extract
+- information
+- liz
+TargetCakeVersion: 0.38.0
+TargetFrameworks:
+- netcoreapp3.1
+- net5.0
+- net6.0
```

Comment:
Hey I'm the author of the addin. What is the issue here?  
I was using the [recommended .NET target version](https://cakebuild.net/docs/extending/addins/best-practices#net-target-version) and the Cake-Version `0.38.0` to be able to support as many users as possible. Do these two not fit together?

Reply:
Hey @wgnf we use major versions of Cake to introduce breaking changes to APIs, so when an addin targets Cake v0.38.0 we can't be sure it will work with Cake v1.0.0 or v2.0.0, or v3.0.0 (latest as of this writing), so your addin should ideally target the most recent major Cake version

You can see more details in the documentation: https://cakebuild.net/docs/extending/addins/best-practices#cake-reference

If you do want to support as many users as possible (and you don't mind the extra work), you should release separae versions of your addin compatible with each Cake major version you want to support.

Reply:
Yeah that totally makes sense (idk what I was thinking there 😅). I'll have a look at that in the next few days.

Reply:
Awesome. Thank you @wgnf!